src/collector/mod.rs

```diff
@@ -19,6 +19,9 @@ use std::collections::HashMap;
 use anyhow::{bail, Result};
 use log::{error, warn};
 
+mod skb;
+use skb::SkbCollector;
```

Comment:
Shouldn't modules be organized the other way around?
i.e: modules create Collectors that `use` the `collector`module